coding/unit_test_snippets/testExecutables.py

```diff
@@ -9,4 +9,5 @@ class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
 UtilsBinaryTester.create_executable_tests(__file__, EXECUTABLES)
 
 if __name__ == "__main__":
+    lsst.utils.tests.init()
```

Comment:
Do we need this? This test script runs executable C++ binaries so I'm assuming it won't inherit the logger setting from the parent python? Or does it? If we do need it then we will also need a `setup_module` in here so that `py.test` will work.